Framework/SwiftOSC/Elements/OSCBundle.swift

```diff
@@ -1,9 +1,18 @@
 import Foundation
 
+
+
+
+
+/// Used to group two or more OSCElements. In most cases, you can bundle two or more OSCMessages to be sent at the same time.
 public class OSCBundle: OSCElement, CustomStringConvertible {
+    
     //MARK: Properties
+    
```

Comment:
A timetag before the current time will be delivered immediately. Bundles with a future timetag will be held in memory and delivered at the correct time. Note, this only works for timetags for Bundles.

Reply:
👍